### ISSUE TITLE
CLI spec setup

### ISSUE THREAD

**Issue by schickling** [prisma/specs#16]
## General page

- [x] Add navigation to individual DB flows
- [x] Add "edit on Github" button @kuldar 

### Nice-to-have
- [x] Way to switch between common Terminal themes
- [ ] Way to switch between common Terminal fonts
- [ ] Way to enable/disable cretain Terminal capabilities (color, character support)

### General

- [ ] Disable scrolling for `Terminal` by default unless specified via `<Terminal scrollable>`
- [ ] Add link component (blocked by https://github.com/xtermjs/xterm.js/issues/1134)
- [ ] Navigation via ink select/enter
- [ ] Fix Progress indicator
- [ ] Adjust terminal window height / width

### Questions

- [x] Enforce indentation for line breaks

### Fix input

- [ ] Multi cursor for text input ![image](https://user-images.githubusercontent.com/1567498/61224009-0bc79f00-a71e-11e9-9313-518e4e6b901d.png)
- [ ] Multi cursor for select ![image](https://user-images.githubusercontent.com/1567498/61224143-547f5800-a71e-11e9-8165-89af4b666153.png)
- [ ] Selection of checkboxes doesn't work
- [ ] White cursor shouldn't be visible unless focussing a text input ![image](https://user-images.githubusercontent.com/1567498/61224234-81cc0600-a71e-11e9-83ef-c2d710f7b9ff.png)
